engine/src/build/config/compiler/BUILD.gn

```diff
@@ -933,6 +933,19 @@ config("no_optimize") {
       cflags += [ "-fomit-frame-pointer" ]
     }
     ldflags = common_optimize_on_ldflags
+  } else if (is_wasm) {
+    # On WASM we don't want to fully deoptimize the debug build because the
+    # compiled binary will have some functions with too many local variables
+    # and Chrome will reject it.
+    cflags = [
+      "-O1",
+      "-fdata-sections",
+      "-ffunction-sections",
+    ]
+    if (!using_sanitizer && !enable_profiling) {
+      cflags += [ "-fomit-frame-pointer" ]
```

Comment:
Does this flag even make sense on wasm? It looks like most of these flags are copypasta'd from the Android one. I'm guessing `-O1` is doing the heavy lifting here, are the other flags needed?